pkg/karmadactl/explain/explain.go

```diff
@@ -85,11 +85,11 @@ func NewCmdExplain(f util.Factory, parentCommand string, streams genericiooption
 	options.AddKubeConfigFlags(flags)
 	options.AddNamespaceFlag(flags)
 	flags.VarP(&o.OperationScope, "operation-scope", "s", "Used to control the operation scope of the command. The optional values are karmada and members. Defaults to karmada.")
-	flags.BoolVar(&o.Recursive, "recursive", o.Recursive, "When true, print the name of all the fields recursively. Otherwise, print the available fields with their description.")
-	flags.StringVar(&o.APIVersion, "api-version", o.APIVersion, "Use given api-version (group/version) of the resource.")
+	flags.BoolVar(&o.ExplainFlags.Recursive, "recursive", o.ExplainFlags.Recursive, "When true, print the name of all the fields recursively. Otherwise, print the available fields with their description.")
+	flags.StringVar(&o.ExplainFlags.APIVersion, "api-version", o.ExplainFlags.APIVersion, "Use given api-version (group/version) of the resource.")
```

Comment:
```suggestion
	o.AddFlags(cmd)
```
The `ExplainFlags` provides the function `AddFlags` for initializing the values of flags.

@RainbowMango Thanks for noticing this~ As the upstream continues to promote the separation of flags and options in the future, bumping Kubernetes dependencies may continue to encounter the same issue.

Reply:
OK, will fix it before next push.

Reply:
Updated as suggested.